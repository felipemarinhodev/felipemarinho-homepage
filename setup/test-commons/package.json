{
  "name": "@felipemarinho/test-commons",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "files": [
    "base.ts"
  ],
  "scripts": {
    "lint": "echo 'Not necessary'",
    "test": "echo 'Not necessary'"
  },
  "devDependencies": {
    "jest": "27.0.0",
    "jest-styled-components": "7.0.8",
    "ts-node": "10.7.0",
    "babel-jest": "28.0.3",
    "@babel/core": "7.17.10",
    "@babel/preset-env": "7.17.10",
    "@babel/preset-react": "7.16.7",
    "@babel/preset-typescript": "7.16.7",
    "@testing-library/react": "13.2.0"
  }
}
