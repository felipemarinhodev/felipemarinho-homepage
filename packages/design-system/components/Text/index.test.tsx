import { Text } from "./index";
import { customRender } from "@felipemarinho/test-commons/react-testing-library";

const render = customRender();

describe("<Textv />", () => {
  it("render h1 tag", () => {
    const { container } = render(<Text tag="h1">Simple Text</Text>);

    expect(container).toMatchSnapshot();
  });
});
