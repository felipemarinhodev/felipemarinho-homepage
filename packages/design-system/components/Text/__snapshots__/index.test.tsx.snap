// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Textv /> render h1 tag 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <h1
    class="c0"
  >
    Simple Text
  </h1>
</div>
`;
